src/bots/depositor.py

```diff
@@ -274,3 +294,48 @@ def _send_deposit_tx(
             self._flashbots_works = True
 
         return success
+
+    def _build_transaction(
```

Comment:
this approach has a problem, if the deposit transaction through mellow falls, we should build vanilla deposit

Reply:
Indeed, didn't handle the case when transaction submission can fail. Reworked it